scripts/format_new.py

```
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+"""
+New implementation of the iMessage formatting tool with cleaner code structure.
+"""
+import argparse
+import asyncio
+import json
+import logging
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional
```

Ruff (0.14.2)

`typing.List` is deprecated, use `list` instead

(UP035)